app/src/test/java/com/x8bit/bitwarden/ui/auth/feature/completeregistration/CompleteRegistrationViewModelTest.kt

```diff
@@ -58,6 +59,11 @@ class CompleteRegistrationViewModelTest : BaseViewModelTest() {
     private val mutableUserStateFlow = MutableStateFlow<UserState?>(null)
     private val mockAuthRepository = mockk<AuthRepository>() {
         every { userStateFlow } returns mutableUserStateFlow
+        coEvery { login(any(), any(), any()) } returns LoginResult.Success
```

Comment:
I guess this is cool as long as the `verify` calls have specific params. I am surprised to not see a `confirmVerified` in here though, which would make me a little more comfortable with the any frenzy. Do you have a general philosophy on when it's cool to `any` all the things? All else being equal the specificity feels more robust to me but that would probably be a nightmare / not at all worth it for this class.

Reply:
Some projects are also far more particular about verifying calls, and I seem to recall Bitwarden not being one of them, so I'm probably thinking too hard about this.

Reply:
so personally I feel like being generic up here is fine since I am setting them globally and then being specific where I am actually testing/calling verify. I will agree the amount of args here are scary looking. 🧟